### PR TITLE
[GITHUB-14] Switched to GoCD source

### PR DESCRIPTION
**Pull request by lobsterdore**
This is so we can install the latest version of the server.

Also bumped the version to 1.1.

### CODE REVIEW

File: tasks/build.yml [hunk 1]
```diff
@@ -22,11 +22,19 @@
     - awscli
     - boto
 
-- name: add GO CD apt repository
+- name: Add GoCD Apt key
+  become: yes
+  apt_key:
+    keyserver: keyserver.ubuntu.com
+    id: 8816C449
```
Comment (racer1680) [inline tasks/build.yml]: Can we move it to defaults?

Comment (lobsterdore) [inline tasks/build.yml]: Yep agreed, I might make this and the source configurable.

File: tasks/build.yml [hunk 2]
```diff
@@ -22,11 +22,19 @@
...
+  apt_key:
+    keyserver: keyserver.ubuntu.com
+    id: "{{ gocd_server.repo.key_id }}"
+    state: present
+
+- name: Install GoCD repo
   become: yes
   apt_repository:
-    repo: deb http://dl.bintray.com/gocd/gocd-deb/ /
+    repo: "{{ gocd_server.repo.repo }}"
```
Comment (racer1680) [inline tasks/build.yml]: How about gocd_server.repo.url ?

**Comment by racer1680** [comment]
LGTM

**Pull request merged**